CIP-XXXX/README.md

```diff
@@ -0,0 +1,141 @@
+---
+CIP: ?
+Title: Orthogonal Certificates
+Status: Proposed
+Category: Meta
+Authors:
+    - Joshua Marchand <josh@securitybot.info>
+Implementors: N/A
+Discussions:
+    - https://discord.gg/tsr7H6ApdT
```

Comment:
```suggestion
    - https://discord.gg/tsr7H6ApdT
    - https://github.com/cardano-foundation/CIPs/pull/1021
```

Reply:
committing to ensure a properly indexed draft for meeting tomorrow.